python/lsst/obs/subaru/SConscript

```diff
@@ -1,5 +1,12 @@
 # -*- python -*-
-from lsst.sconsUtils import scripts
-scripts.BasicSConscript.pybind11(["crosstalk",
-                                  "hscDistortion"])
+from lsst.sconsUtils import env, targets
 
+# Cannot use here?
```

Comment:
I dunno if you can use it here or not, but if you can't you don't want it here-but-commented. :-)